### PR TITLE
selenium-manager : Add Version 1.0.0-M3

### PR DESCRIPTION
**Pull request by Baneeishaque**
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/selenium-manager.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "version": "1.0.0-M3",
+    "description": "Selenium Manager is a CLI (Command-Line Interface) tool developed in Rust to allow cross platform execution. On its beta 1 version, Selenium Manager is invoked transparently by the Selenium bindings when no browser driver is detected on the PATH or no third party driver manager is being used on the system.",
```
Comment (rashil2000) [inline bucket/selenium-manager.json:3]:
```suggestion
    "description": "Selenium Manager is a command-line tool developed in Rust to allow cross platform execution.",
```

File: bucket/selenium-manager.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "homepage": "https://www.selenium.dev/blog/2022/introducing-selenium-manager",
+    "license": "Apache-2.0",
+    "notes": "Check selenium-manager --help for usages, example : selenium-manager --browser chrome",
```
Comment (rashil2000) [inline bucket/selenium-manager.json:6]:
```suggestion
```
This is not required

**Review by rashil2000 — CHANGES_REQUESTED** [review]
This is a 64bit application, so the architecture block is needed.

**Comment by Baneeishaque** [comment]
I appreciated your changes. I was little busy - that's why i can't respond in time.
Anyway, thank you for your corrections & commits.

Just because of curiosity, How do you find that binary is 64-bit only

**Comment by rashil2000** [comment]
> Just because of curiosity, How do you find that binary is 64-bit only?

Using a program called `file`: https://www.darwinsys.com/file/

**Pull request merged**